Sprinkle of Yeezus/Settings.swift

```diff
@@ -11,52 +11,68 @@ import UIKit
 import UserNotifications
 
 class SettingsPage: UIViewController {
-    
 
-    @IBOutlet weak var InfoLabel: UILabel!
-    @IBOutlet weak var DescButton: UIButton!
-    @IBOutlet weak var DescLabel: UILabel!
-    @IBOutlet weak var Switch: UISwitch!
+    @IBOutlet private weak var InfoLabel: UILabel!
+    @IBOutlet private weak var DescButton: UIButton!
+    @IBOutlet private weak var DescLabel: UILabel!
+    @IBOutlet private weak var Switch: UISwitch!
+    @IBOutlet private weak var TimePicked: UIDatePicker!
     
-    var notificationsOn = Bool()
+    private var sprinkleTimePicked = Date()
```

Comment:
You'll notice that I moved this variable to the top for starters. A lot of coders have different preferences for file structure. Here is how I normally order my properties and functions in my files, and you can decide if you like it or if you would like to make changes. Either way, we should adopt an order that you can use throughout.

1. Publicly available IBOutlets
2. Private IBOutlets
3. Public variables
4. Private variables
5. Lifecycle functions (`viewDidLoad`, `viewWillAppear`, etc.)
6. IBActions (Which for the most part are always private
7. Public shared functions
8. Private functions

And then any delegate I'm implementing I put in a separate extension after the main class. For instance, if I have a UITextField and I need a HomeViewController class to adapt the UITextFieldDelegate protocol, I add the following after the main class:

```
extension HomeViewController: UITextFieldDelegate {
    // Implement protocol here
}
```